### PR TITLE
[xray] Add support for global values

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
[xray] Add support for global values

### CODE REVIEW

File: stable/xray/values.yaml [hunk 1]
```diff
@@ -43,8 +120,8 @@ xray:
     shared:
       logging:
         consoleLog:
-          enabled: {{ .Values.xray.consoleLog }}
-      jfrogUrl: "{{ tpl (required "\n\nxray.jfrogUrl is required! This allows to connect to Artifactory.\n(You can copy the JFrog URL from Admin > Security > Settings)" .Values.xray.jfrogUrl) . }}"
+          enabled: {{ .Values.global.consoleLog | default .Values.xray.consoleLog }}
```
Comment (danielezer) [inline stable/xray/values.yaml:123]: Let’s keep it consistent. Create a function for this as well

**Review by danielezer — CHANGES_REQUESTED** [review]

File: stable/xray/values.yaml (hunk 1 above)
Comment (chukka) [reply stable/xray/values.yaml]: Done

**Review by danielezer — APPROVED** [review]

File: stable/xray/ci/global-section.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+unifiedUpgradeAllowed: true
```
Comment (chukka) [inline stable/xray/ci/global-section.yaml:1]: @rahulsadanandan  can you append the file name `global-section-values.yaml`

Comment (rahulsadanandan) [reply stable/xray/ci/global-section.yaml]: Changed

File: stable/xray/values.yaml [hunk 3]
```diff
@@ -43,8 +105,8 @@ xray:
     shared:
       logging:
         consoleLog:
-          enabled: {{ .Values.xray.consoleLog }}
-      jfrogUrl: "{{ tpl (required "\n\nxray.jfrogUrl is required! This allows to connect to Artifactory.\n(You can copy the JFrog URL from Admin > Security > Settings)" .Values.xray.jfrogUrl) . }}"
+          enabled: {{ include "xray.consoleLog" . }}
```
Comment (eldada) [inline stable/xray/values.yaml:108]: I think this should be false by default for all helm charts. There's no value for it in k8s. It just duplicates the Docker's stdout.

Comment (rahulsadanandan) [reply stable/xray/values.yaml:108]: By default `xray.consoleLog` is set to false in values.yaml .

**Review by rahulsadanandan — APPROVED** [review]

**Review by rimusz — APPROVED** [review]
LGTM

**Pull request merged**